zhaquirks/philips/soc001.py

```diff
@@ -0,0 +1,45 @@
+"""Signify SOC001 device."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import BinarySensorDeviceClass, ClusterType, QuirkBuilder
+
+
+class PhilipsContactCluster(CustomCluster):
+    """Philips manufacturer specific cluster for contact sensor."""
+
+    cluster_id = 64518  # 0xfc06
+    name = "Philips contact cluster"
+    ep_attribute = "philips_contact_cluster"
+    attributes = {
+        0x0100: ("contact", types.enum8, True),
+        0x0101: ("last_contact_change", types.uint32_t, True),
+        0x0102: ("tamper", types.enum8, True),
+        0x0103: ("last_tamper_change", types.uint32_t, True),
+    }
+
+
+(
+    #  <SimpleDescriptor endpoint=2 profile=260 device_type=1026
+    #  device_version=0
+    #  input_clusters=[0, 1, 3, 64518]
+    #  output_clusters=[0, 3, 6, 25]>
+    QuirkBuilder("Signify Netherlands B.V.", "SOC001")
+    .removes(cluster_id=6, endpoint_id=2, cluster_type=ClusterType.Client)
+    .replaces(
+        replacement_cluster_class=PhilipsContactCluster, cluster_id=64518, endpoint_id=2
+    )
+    .binary_sensor(
+        attribute_name="contact",
+        cluster_id=64518,  # 0xfc06
+        endpoint_id=2,
+        device_class=BinarySensorDeviceClass.OPENING,
+    )
+    .binary_sensor(
+        attribute_name="tamper",
+        cluster_id=64518,  # 0xfc06
+        endpoint_id=2,
+        device_class=BinarySensorDeviceClass.TAMPER,
+    )
```

Comment:
We may also want to include `fallback_name`. It will be supported in future HA versions. If the translation key isn't in HA yet (probably not the case until a ZHA bump with the new quirk is merged to HA), it will fall back to that name and ignore the translation key.

And I'm not sure what the current state on this is:
Currently, `translation_key` defaults to `attribute_name` if not explicitly provided, but it might be required in future versions(?) Again, not sure what we set on. Both might still work.

Reply:
Also, we could also drop `attribute_name` and `cluster_id` here, since they're always the first two arguments:
```python
    .binary_sensor(
        "tamper",
        PhilipsContactCluster.cluster_id,
        endpoint_id=2,
        device_class=BinarySensorDeviceClass.TAMPER,
    )
```